### DISCUSSION TITLE
How to apply for Github Education? (for manual review)

### DISCUSSION THREAD

**Discussion by DreamCake123** [community/community#111061] [GitHub Education] [answered]
### Hi everyone,

so I got rejected some time ago by Github Edu because I did not put my real name on my profile. Now I want to re-apply again but it seems like Github Edu cannot recognize my institution now.
When I upload my documents e.g. student ID, transcript, payment advice, enrolment letter, then click process application, it just keeps redirecting me to the same page regardless of what I upload.
I have searched this issue for quite some time online, found other posts but doesn't seem to lead anywhere. All posts are dead and not solved.
Is there a way I could contact a support for manual review of my application? Or is there any chance they are going to fix this issue of the website/application process?

**Comment by DanielFreitassc** [comment]
Try using an email from the institution and linking it to your main profile.

**Reply by DreamCake123** [reply]
I have put the info in my emails section under my profile. Now it prompts me to join Github Education.
But when I click process application after I've submitted my photo proofs, it still redirects me to the same page and needs me to upload again without any notifications for errors or anything.

**Reply by 2ssunny** [reply]
In my case, I have same problem also. Actually, my school doesn't provide email for students, so I need to prove with the proof letter of attendance. But, because of this problem, I can not upload my document. Is this issue solved for you?

**Reply by hobi-design** [reply]
Same problem here. Created a ticket about three weeks ago, yet to get anywhere. I tried applying with my enrollment form (confirming my admission) that has the same email address I am using here but still no bueno.

**Reply by winds09** [reply]
I met the same problem when I used my iPhone to apply. But if I apply with an iPad or a PC, the problem can be solved.

**Comment by samus-aran** [comment]
Hi everyone!

Are you able to provide more details regarding to when you submit your evidence the page refreshes:

- Console log errors (via your Browser developer tools)
- Screen-recording or screenshots

This will help us troubleshoot the issue you're experiencing. 👍

**Reply by DreamCake123** [reply]
Exactly same situation as yours. Though remember to black out your personal details, just to be safe.

**Comment by taylorjeandev** [comment, marked as answer]
I was having this same issue and resolved it by reducing the photo size. Although it says the limit is 1mb, the limit is less than that. My limit was 712kb and i got an error saying the 'payload was too big'.

Reduced the size to just over 100kb, and my application got processed. Hope this helps.

**Reply by DreamCake123** [reply]
Thank you. The request is now processing.
It is kind of funny that nobody at github noticed this.
